test/test_client.py

```diff
@@ -535,6 +535,16 @@ def test_client_options(self):
         self.assertIsInstance(c.options.retry_writes, bool)
         self.assertIsInstance(c.options.retry_reads, bool)
 
+    def test_validate_suggestion(self):
+        """Validate kwargs in constructor."""
+        self.assertRaises(ConfigurationError, MongoClient, auth="standard")
+
+        try:
+            MongoClient(auth="standard")
```

Comment:
Could you also add a test for a camelCase option to ensure we report the original non-lowercased name?

Reply:
Still todo.